src/components/Image/Image.mdx

```diff
@@ -107,8 +107,17 @@ in an element with a fixed width value.
 
 ## HTML Attributes
 
-The `Image` component accepts HTML image attributes. If needed, you can pass
-functions such as `onLoad` and `onError` or attributes such as `srcset`.
+The `Image` component accepts HTML image attributes, and in fact, if you are
+using the internal DS image (not passing a custom image component),
+you must pass an HTML `src` attribute.
+
+Additionally, unless the image is purely decorative, the `alt` attribute is
+also required to ensure the image is accessible. If the image is purely
+decorative, you may skip passing an `alt` attribute explictly as `alt=""`
+will automatically be applied.
+
+If needed, you can pass functions such as `onLoad` and `onError` or attributes
```

Comment:
This is true but I'm not sure if or how we support `srcset`.

Reply:
Me neither. This was already here, I just moved it slightly. Would you like me to remove?